Bottom: 04d575518b15337c20d20d1731d796dd81bd05c3
Top:    8adef1560a07205792c6f021a1d92ec6764387f0
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-22 13:48:57 -0500

Refresh of TESTING--VirtualMachine

---
diff --git a/vmdaedalus/vagrant_ansible_inventory b/vmdaedalus/vagrant_ansible_inventory
new file mode 100644
index 0000000..c6b5207
--- /dev/null
+++ b/vmdaedalus/vagrant_ansible_inventory
@@ -0,0 +1,14 @@
+# Generated by Vagrant
+
+vmdaedalus.novatech-llc.com network_connection="System enp0s8" ansible_host=127.0.0.1 ansible_port=2222 ansible_user='vagrant' ansible_ssh_private_key_file='/home/coopera/Developer/testdaedalus/.vagrant/machines/vmdaedalus.novatech-llc.com/virtualbox/private_key'
+buildbot-worker.novatech-llc.com ansible_host=127.0.0.1 ansible_port=2200 ansible_user='vagrant' ansible_ssh_private_key_file='/home/coopera/Developer/testdaedalus/.vagrant/machines/buildbot-worker.novatech-llc.com/virtualbox/private_key'
+
+[bacula]
+vmdaedalus.novatech-llc.com
+
+[docker-hosts]
+vmdaedalus.novatech-llc.com
+
+[buildbot]
+vmdaedalus.novatech-llc.com
+buildbot-worker.novatech-llc.com
